bluesky_config/ipython/profile_default/startup/00-base.py

```diff
@@ -35,7 +33,7 @@
 zmq_publisher = zmqPublisher("127.0.0.1:4567")
 kafka_publisher = kafkaPublisher(
     topic="mad.bluesky.documents",
-    bootstrap_servers="127.0.0.1:9092",
+    bootstrap_servers="127.0.0.1:29092",
```

Comment:
Is the port a typo?

Reply:
No, 

```
-e KAFKA_CFG_ADVERTISED_LISTENERS=PLAINTEXT://localhost:29092,PLAINTEXT_HOST://$IP_ADDR:9092 \
```

says that if you connect via port 29092 you get told that the address is `localhost:29092` and if you come in on 9092 you get told the address is `192.168.80.36:9092` (or what ever the IP of the host machine is).   In this case because we are running inside the pod we want to connect to 29092 to make sure we don't go out of the pod if we don't need to.